change-address/LETTERSPECS/BANNO.CHANGE.ADDR.V1.CFG

```diff
@@ -79,11 +97,11 @@ PR: YES
 *  VALID VALUES: You can declare up to 20 different character elements to check for.
 * (PO Box List 01 through PO Box List 20)
 *  DEFAULT: (there is no default, valid entry is required)
-PO Box List 01: P.O.BOX
-PO Box List 02: P.O. BOX
-PO Box List 03: PO BOX
-PO Box List 04: PO. BOX
-PO Box List 05: POST OFFICE BOX
+PO Box List 01: PO BOX
```

Comment:
Is there a reason the Previous P.O. BOX , etc. values were removed? My guess if having the most standard values pre filled has been helpful to CUs

Reply:
An oversight @jordan-keenan - I'll add them back in.